tests/countries/test_guadeloupe.py

```
@@ -0,0 +1,165 @@
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from unittest import TestCase
+
+from holidays.countries.guadeloupe import HolidaysGP, Guadeloupe, GP, GLP
+from tests.common import CommonCountryTests
+
+
+class TestMartinique(CommonCountryTests, TestCase):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)